config/v1/types_feature.go

```diff
@@ -126,6 +126,7 @@ var defaultFeatures = &FeatureGateEnabledDisabled{
 		"NodeDisruptionExclusion",        // sig-scheduling, ccoleman
 		"ServiceNodeExclusion",           // sig-scheduling, ccoleman
 		"SCTPSupport",                    // sig-network, ccallend
+		"DownwardAPIHugePages",           // sig-node, bmcfall
```

Comment:
This is a node feature.  It needs a name and an ack from the node team to enable this by default.